examples/async_single_threaded.rs

```diff
@@ -74,8 +74,12 @@ fn main() {
                             .await;
                         let session = session.unwrap();
                         let session2 = session2.unwrap();
-                        web_socket.connect(&session, true, -1).await;
-                        web_socket2.connect(&session2, true, -1).await;
+                        web_socket
```

Comment:
Why does this accept a port string now? There shouldn't be cases where the socket connects to a port that is different from the client.